modules/installing-rhv-setting-up-ca-certificate.adoc

```diff
@@ -16,16 +16,16 @@ Later, you provide the certificate to the installation program.
 .Procedure
 
 . Use either of these two methods to download the CA certificate:
-** Go to the Manager's webpage, `\https://<engine-fqdn>/ovirt-engine/`. Then, under *Downloads*, click the *CA Certificate* link.
+** Go to the {rh-virtualization-engine-name}'s webpage, `\https://<{rh-virtualization-engine-name}-fqdn>/ovirt-engine/`. Then, under *Downloads*, click the *CA Certificate* link.
 ** Run the following command:
 +
 [source,terminal]
 ----
-$ curl -k 'https://<engine-fqdn>/ovirt-engine/services/pki-resource?resource=ca-certificate&format=X509-PEM-CA' -o /tmp/ca.pem  <1>
+$ curl -k 'https://<{rh-virtualization-engine-name}-fqdn>/ovirt-engine/services/pki-resource?resource=ca-certificate&format=X509-PEM-CA' -o /tmp/ca.pem  <1>
```

Comment:
Appears as: 
`curl -k 'https://<{rh-virtualization-engine-name}-fqdn>/ovirt-engine/services/pki-resource?resource=ca-certificate&format=X509-PEM-CA' -o /tmp/ca.pem  `

Reply:
Changed to <engine-fqdn>